### ISSUE TITLE
[BUG] bad syntax highlighting

### ISSUE THREAD

**Issue by thjazi39** [Felienne/hedy#1934]
**Describe the bug**
Syntactic coloring does not take the level into account.

**Add a screenshot (optional)**

![image](https://user-images.githubusercontent.com/48509466/153391957-d25131a9-cd8c-46db-adc5-e2d8e8224efe.png)

**Expected behavior**
In the example above, we can see that at level 4, there is syntactic coloring on the text between quotation marks, this is a problem because at level 4, you should not put quotation marks on the text.
You should either color the text even when it is not in quotation marks, or remove the coloring for the text in quotation marks.
This should prevent the children from wondering about the text that appears in a different color.

it is possible that there are other errors of this type in other levels. in any case, syntax highlighting could be an additional tool to help children understand the different elements of the code (for example, differentiate variables from text)

**Comment by Felienne** [comment]
Yes, we know, this is an open issue that we are already working on #1285. Please check this before you open an issue (can be done in Slack too if you are in doubt), and also refrain from using negative words like "bad", it suffices to say: "missing", or "an issue with".
